src/dmd/parse.d

```diff
@@ -900,7 +914,8 @@ final class Parser(AST) : Lexer
                     a = parseBlock(pLastDecl, pAttrs);
                     if (pAttrs.udas)
                     {
-                        s = new AST.UserAttributeDeclaration(pAttrs.udas, a);
+                        // [3] int a = 3;
```

Comment:
`// @(3)`?

Reply:
No, the parser recognizes the old syntax, `[3]`, for UDAs, which was never officially released.

Reply:
Huh, the more you know.